test/update_test.go

```diff
@@ -311,6 +378,76 @@ var _ = Describe("update", func() {
 				})
 			})
 		})
+
+		Context("--as-workflow flag", func() {
```

Comment:
Is this supposed to be inside the `Context("dependencies.yaml does not exist", ...)` on line 336?

Reply:
i don't think so